bin/mark_changepoints_in_json

```diff
@@ -174,27 +176,28 @@ class Segments(object):
                 self._merge_segment(0)
 
     def get_classification(self):
-        """Return the classification for this run sequence.
-        """
-        classification = ''
-        # Flat -> no changepoints.
-        if len(self.segments) == 1:
-            classification = 'flat'
-        # No steady state -> there are changes after we expect steady state.
-        elif self.segments[-1].start > (self.length - self.steady_state):
-            classification = 'no steady state'
-        # Warmup -> first segment mean is highest and last is the lowest.
-        elif (self.segments[-1].start <= (self.length - self.steady_state) and
-              self.segments[-1].mean == min(self.segments, key=lambda s: s.mean).mean):
+        """Return a classification for this run sequence."""
```

Comment:
Sorry, I misunderstood you. I stupidly thought you meant a blank line within the docstring... which was a silly thing for me to think. Mea culpa!